### PR TITLE
feat(application integration): Add example for editing region

### PR DESCRIPTION
**Pull request by pay20**
Prepared application integration terraform sample for inclusion in C.G.C. documentation

## Description

Intending to use this snippet on the following page:
https://cloud.google.com/application-integration/docs/enable-new-region

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [ *] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [ *] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [* ] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [ *] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [ *] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [ *] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [* ] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s): https://cloud.google.com/application-integration/docs/enable-new-region

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [ ] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: applicationintegration/applicationintegration_edit_region/main.tf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  location = "us-central1"
+}
+
+resource "google_kms_crypto_key" "cryptokey" {
+  name            = "crypto-key-example"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "7776000s"
+  depends_on      = [google_kms_key_ring.keyring]
+}
+# [START applicationintegration_edit_region]
```
Comment (iennae) [inline applicationintegration/applicationintegration_edit_region/main.tf:30]: is this the right region tag prefix?

Comment (iennae) [reply applicationintegration/applicationintegration_edit_region/main.tf:30]: just verified, it should be application_integration

Comment (pay20) [reply applicationintegration/applicationintegration_edit_region/main.tf:30]: Thanks! I've fixed the region tag in https://github.com/terraform-google-modules/terraform-docs-samples/pull/637/commits/98d1c3103c72bc44bf40cd0d5e10d91f18ab181d.

**Comment by iennae** [comment]
@pay20 one of your commits you didn't sign appropriately so this is what is breaking the CLA check. Additionally it looks like you still need to update the region tag as snippet-bot-check is still failing.

**Comment by pay20** [comment]
Hi @iennae
I'm closing this PR. I've added changes in a new PR [new pr#646](https://github.com/terraform-google-modules/terraform-docs-samples/pull/646). Can you PTAL? Thanks for the help!

**Pull request closed without merging**